manim/animation/indication.py

```diff
@@ -41,6 +41,7 @@ def construct(self):
 
 import numpy as np
 from colour import Color
+from deprecated import deprecated
```

Comment:
Hello, I think the reason the docs are failing is that we did not install `Python-Deprecated` package.
`pip install Python-Deprecated`

**Reference:** https://python-deprecated.readthedocs.io/en/latest/installation.html

Reply:
https://docs.manim.community/en/stable/reference/manim.utils.deprecation.html?highlight=deprecation

We have our own in-house `deprecation` module, we don't use an external package. You can import it with 
`from ..utils.deprecation import deprecated`

You can also search through the repo for other instances where it is used for more clarification

Reply:
sure👍